tsql/TSqlParser.g4

```diff
@@ -1950,14 +1959,17 @@ create_statistics
       (WITH (FULLSCAN | SAMPLE DECIMAL (PERCENT | ROWS) | STATS_STREAM)
             (',' NORECOMPUTE)? (',' INCREMENTAL EQUAL on_off)? )? ';'?
     ;
+update_statistics
+    : UPDATE (INDEX|ALL)? STATISTICS full_table_name id?  (USING DECIMAL VALUES)?
+    ;
 
 // https://msdn.microsoft.com/en-us/library/ms174979.aspx
 create_table
-    : CREATE TABLE table_name '(' column_def_table_constraints ','? ')' table_options* (ON id | DEFAULT)? (TEXTIMAGE_ON id | DEFAULT)?';'?
+    : CREATE TABLE table_name '(' column_def_table_constraints ','? ')' (LOCK simple_id)? table_options* (ON id | DEFAULT)? (TEXTIMAGE_ON id | DEFAULT)?';'?
     ;
 
 table_options
-    : WITH '(' index_option (',' index_option)* ')'
+    : WITH '('? index_option (',' index_option)* ')'?
```

Comment:
Correct record: `WITH ('(' index_option (',' index_option)* ')' | index_option (',' index_option)*)`. Single parenthesis is not correct in lanugage but allowed in your rule.

Reply:
The paranthesis are optional in SYBASE TSQL, which is the dialect I’m using.

Reply:
Expression with only one parenthesis is allowed? I'm sure this is incorrect: `WITH '(' index_option (',' index_option)*`. Check my sentence more carefully.